src/app/core/models/user.model.ts

```diff
@@ -1,6 +1,13 @@
 export interface User {
```

Comment:
can you please add property "avatar" which includes a link.  We can get a user avatar by using the link from firebase storage.

Reply:
or it can be named avatarUrl for example